### PR TITLE
Fix post release PR step

### PR DESCRIPTION
**Pull request by TharmiganK**

### CODE REVIEW

**Review by lnash94 — APPROVED** [review]

File: .github/workflows/publish-ballerina-to-openapi.yml [hunk 1]
```diff
@@ -44,5 +44,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
-          curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
-          bin/hub pull-request -m "[Automated] Sync master after ballerina-to-openapi:$VERSION release"
+          git checkout -b ballerina-to-openapi:$VERSION
```
Comment (TharmiganK) [inline .github/workflows/publish-ballerina-to-openapi.yml:47]:
```suggestion
          git checkout -b ballerina-to-openapi-release:$VERSION
```

File: .github/workflows/publish-ballerina-to-openapi.yml [hunk 2]
```diff
@@ -44,5 +44,6 @@ jobs:
...
+          git push origin ballerina-to-openapi:$VERSION
```
Comment (TharmiganK) [inline .github/workflows/publish-ballerina-to-openapi.yml:48]:
```suggestion
          git push origin ballerina-to-openapi-release:$VERSION
```

**Review by lnash94 — DISMISSED** [review]

File: .github/workflows/publish-ballerina-to-openapi.yml [hunk 3]
```diff
@@ -44,5 +44,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
-          curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
-          bin/hub pull-request -m "[Automated] Sync master after ballerina-to-openapi:$VERSION release"
+          git push origin ballerina-to-openapi-release:${VERSION}
```
Comment (TharmiganK) [inline .github/workflows/publish-ballerina-to-openapi.yml:47]:
```suggestion
          git push origin ballerina-to-openapi-release-${VERSION}
```

File: .github/workflows/publish-ballerina-to-openapi.yml [hunk 4]
```diff
@@ -44,5 +44,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
-          curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
-          bin/hub pull-request -m "[Automated] Sync master after ballerina-to-openapi:$VERSION release"
+          git push origin ballerina-to-openapi-release-${VERSION}
```
Comment (TharmiganK) [inline .github/workflows/publish-ballerina-to-openapi.yml:47]:
```suggestion
          git push origin .
```

**Review by lnash94 — APPROVED** [review]

**Comment by TharmiganK** [comment]
Closing in favour of https://github.com/ballerina-platform/openapi-tools/pull/1758

**Pull request closed without merging**